### ISSUE TITLE
Sides seem to be switched

### ISSUE THREAD

**Issue by bdmendes** [Disservin/fast-chess#133]
```
Score of camel-rework-mate-in-plys vs camel-master: 1 - 0 - 1 [] 2
Elo difference: 190.85 +/- nan, LOS: 84.13 %, DrawRatio: 50.00 %
Ptnml:        WW     WD  DD/WL     LD     LL
Distr:         0      1      0      0      0
--------------------------------------------------
Started game 6 of 800 (camel-master vs camel-rework-mate-in-plys)
Finished game 1 (camel-rework-mate-in-plys vs camel-master): 0-1 {camel-rework-mate-in-plys got checkmated}
Started game 7 of 800 (camel-rework-mate-in-plys vs camel-master)
Finished game 2 (camel-master vs camel-rework-mate-in-plys): 1-0 {camel-rework-mate-in-plys got checkmated}
--------------------------------------------------
Score of camel-rework-mate-in-plys vs camel-master: 3 - 0 - 1 [] 4
Elo difference: 338.04 +/- nan, LOS: 95.84 %, DrawRatio: 25.00 %
Ptnml:        WW     WD  DD/WL     LD     LL
Distr:         1      1      0      0      0
--------------------------------------------------
Started game 8 of 800 (camel-master vs camel-rework-mate-in-plys)
Finished game 5 (camel-rework-mate-in-plys vs camel-master): 0-1 {camel-rework-mate-in-plys got checkmated}
Started game 9 of 800 (camel-rework-mate-in-plys vs camel-master)
Finished game 6 (camel-master vs camel-rework-mate-in-plys): 1/2-1/2 {Draw by 3-fold repetition}
--------------------------------------------------
Score of camel-rework-mate-in-plys vs camel-master: 4 - 0 - 2 [] 6
Elo difference: 279.59 +/- nan, LOS: 97.72 %, DrawRatio: 33.33 %
Ptnml:        WW     WD  DD/WL     LD     LL
Distr:         1      2      0      0      0
--------------------------------------------------
Started game 10 of 800 (camel-master vs camel-rework-mate-in-plys)
Finished game 7 (camel-rework-mate-in-plys vs camel-master): 0-1 {camel-rework-mate-in-plys got checkmated}
Started game 11 of 800 (camel-rework-mate-in-plys vs camel-master)
Finished game 8 (camel-master vs camel-rework-mate-in-plys): 1-0 {camel-rework-mate-in-plys got checkmated}
--------------------------------------------------
Score of camel-rework-mate-in-plys vs camel-master: 6 - 0 - 2 [] 8
Elo difference: 338.04 +/- nan, LOS: 99.28 %, DrawRatio: 25.00 %
Ptnml:        WW     WD  DD/WL     LD     LL
Distr:         2      2      0      0      0
--------------------------------------------------
```

**Comment by Disservin** [comment]
What am I supposed to see?😅

**Comment by bdmendes** [comment]
The new version is being checkmated and the elo difference is increasing (positively) as it loses games: I only got this once though

**Comment by Disservin** [comment]
That might be the case with this reporting style, the elo report is done whenever a pair has been played so a vs b and b vs a.

**Comment by bdmendes** [comment]
I understand, but I assumed the scores were Engine1Win - Engine2Win - Draw, right?

**Comment by Disservin** [comment]
Mhh game 1 and game 2 should be in one group and both times  master won but the update was wrong, now I get it… this looks indeed wrong. I’ll check tomorrow

**Comment by bdmendes** [comment]
There is definitely something wrong, my old non-bitboard version is getting some ghost wins in the score
```
Starting tournament...
Started game 1 of 400 (camel-gauntlet-books vs camel-v0.3.0)
Started game 3 of 400 (camel-gauntlet-books vs camel-v0.3.0)
Started game 2 of 400 (camel-v0.3.0 vs camel-gauntlet-books)
Started game 4 of 400 (camel-v0.3.0 vs camel-gauntlet-books)
Finished game 3 (camel-gauntlet-books vs camel-v0.3.0): 1-0 {camel-v0.3.0 got checkmated}
Started game 5 of 400 (camel-gauntlet-books vs camel-v0.3.0)
Finished game 5 (camel-gauntlet-books vs camel-v0.3.0): 1-0 {camel-v0.3.0 got checkmated}
Started game 6 of 400 (camel-v0.3.0 vs camel-gauntlet-books)
Finished game 2 (camel-v0.3.0 vs camel-gauntlet-books): 0-1 {camel-v0.3.0 got checkmated}
Started game 7 of 400 (camel-gauntlet-books vs camel-v0.3.0)
Finished game 1 (camel-gauntlet-books vs camel-v0.3.0): 1-0 {camel-v0.3.0 got checkmated}
--------------------------------------------------
Score of camel-gauntlet-books vs camel-v0.3.0: 2 - 0 - 0 [] 4
Elo difference: inf +/- -nan, LOS: 92.14 %, DrawRatio: 0.00 %
Ptnml:        WW     WD  DD/WL     LD     LL
Distr:         1      0      0      0      0
--------------------------------------------------
Started game 8 of 400 (camel-v0.3.0 vs camel-gauntlet-books)
Finished game 4 (camel-v0.3.0 vs camel-gauntlet-books): 0-1 {camel-v0.3.0 got checkmated}
--------------------------------------------------
Score of camel-gauntlet-books vs camel-v0.3.0: 2 - 2 - 0 [] 5
Elo difference: -0.00 +/- 786.22, LOS: 50.00 %, DrawRatio: 0.00 %
Ptnml:        WW     WD  DD/WL     LD     LL
Distr:         1      0      0      0      1
--------------------------------------------------
Started game 9 of 400 (camel-gauntlet-books vs camel-v0.3.0)
Finished game 8 (camel-v0.3.0 vs camel-gauntlet-books): 0-1 {camel-v0.3.0 got checkmated}
Started game 10 of 400 (camel-v0.3.0 vs camel-gauntlet-books)
Finished game 6 (camel-v0.3.0 vs camel-gauntlet-books): 1/2-1/2 {Draw by 3-fold repetition}
--------------------------------------------------
Score of camel-gauntlet-books vs camel-v0.3.0: 2 - 3 - 1 [] 7
Elo difference: -58.45 +/- 349.50, LOS: 32.74 %, DrawRatio: 16.67 %
Ptnml:        WW     WD  DD/WL     LD     LL
Distr:         1      0      0      1      1
```

**Comment by bdmendes** [comment]
Thank you!

**Comment by Disservin** [comment]
should be fixed in tag `v0.6.2-alpha`
